backend/development.ini.sample

```diff
@@ -364,15 +364,10 @@ api.key = %(basic_setup.api_key)s
 
 ## User custom properties: This features allows defining custom instance-global fields on users of this instance.
 # See the Tracim settings documentation (available in source code, see "user_custom_properties.md") for more
-# information about it. By default, Tracim doesn't use custom properties.
```

Comment:
Need changeor remove wording: ` By default, Tracim doesn't use custom properties.`

Reply:
This is removed, you are looking at a replaced line :-)

Reply:
Good.